SPECS/qtbase/qtbase.spec

```diff
@@ -217,79 +174,34 @@ Requires: %{name}-devel%{?_isa} = %{version}-%{release}
 
 # debating whether to do 1 subpkg per library or not -- rex
 %package gui
-Summary: Qt5 GUI-related libraries
-Obsoletes: qt5-qtbase-x11 < 5.2.0
-Provides:  qt5-qtbase-x11 = %{version}-%{release}
-# for Source6: 10-qt5-check-opengl2.sh:
-# glxinfo
-#Requires: glx-utils
+Summary: Qt GUI-related libraries
+Provides:  qtbase-x11 = %{version}-%{release}
 %description gui
-Qt5 libraries used for drawing widgets and OpenGL items.
-
+Qt libraries used for drawing widgets and OpenGL items.
 
 %prep
-%setup -q -n %{qt_module}-everywhere-src-%{version}
-
-## upstream fixes
-
-%patch 4 -p1 -b .QTBUG-35459
-# omit '-b .tell-the-truth-about-private-api' so it doesn't end up in installed files -- rdieter
-%patch 8 -p1
-
-%patch 50 -p1 -b .QT_VERSION_CHECK
-%patch 51 -p1 -b .hidpi_scale_at_192
-%patch 52 -p1 -b .moc_macros
-%patch 53 -p1 -b .qt5gui_cmake_isystem_includes
-%patch 54 -p1 -b .qmake_LFLAGS
-%patch 61 -p1 -b .qt5-qtbase-cxxflag
-%patch 64 -p1 -b .firebird
-%patch 65 -p1 -b .mysql
-%patch 68 -p1
-
-%patch 80 -p1 -b .use-wayland-on-gnome.patch
-%patch 81 -p1
-%patch 82 -p1
-%patch 83 -p1
-%patch 84 -p1
-%patch 86 -p1
-%patch 87 -p1
-
-## upstream patches
-
+%autosetup -n %{qt_module}-everywhere-src-%{version} -p1
 
 # move some bundled libs to ensure they're not accidentally used
 pushd src/3rdparty
 mkdir UNUSED
-mv freetype libjpeg libpng zlib UNUSED/
-%if "%{?sqlite}" == "-system-sqlite"
-mv sqlite UNUSED/
-%endif
+mv harfbuzz-ng freetype libjpeg libpng zlib UNUSED/
```

Comment:
I'm wondering can we just remove them instead of creating an UNUSED folder and moving them there?

Also curious how freetype is satisfied.  Doesn't seem like a requires above.

Reply:
Changed to remove these folders instead